src/test/FuzzerImpl.cpp

```diff
@@ -913,11 +1129,25 @@ TransactionFuzzer::xdrSizeLimit()
     return 50 * FuzzUtils::FUZZER_MAX_OPERATIONS;
 }
 
+void
+TransactionFuzzer::createAppAndRootAccount()
+{
+    resetRandomSeed(1);
+    mApp = createTestApplication(mClock, getFuzzConfig(0));
+    mRoot = std::make_shared<TestAccount>(TestAccount::createRoot(*mApp));
+    mSourceAccountID = mRoot->getPublicKey();
+    resetTxInternalState(*mApp);
+}
+
 #define FUZZER_INITIAL_CORPUS_OPERATION_GEN_UPPERBOUND 128
 void
 TransactionFuzzer::genFuzz(std::string const& filename)
 {
-    gRandomEngine.seed(std::random_device()());
+    createAppAndRootAccount();
+
+    LedgerTxn ltxOuter(mApp->getLedgerTxnRoot());
```

Comment:
Correct, that was only going to be necessary for the `genFuzz` valid-transaction change.  Since it's not obvious to me that that one is actually helping (I've stashed the code in a private branch and might fiddle with it if I get a chance to see if I can make it useful), I've removed it from this PR, and thus `createAppAndRootAccount()` is gone too.